app/gui/src/main/scala/org/bitcoins/gui/ContractGUI.scala

```diff
@@ -0,0 +1,274 @@
+package org.bitcoins.gui
+
+//import akka.actor.ActorSystem
+import org.bitcoins.cli.CliCommand.{
+  AcceptDLCCliCommand,
+  AddDLCSigsAndBroadcastCliCommand,
+  CreateDLCOffer,
+  SignDLCCliCommand
+}
+import org.bitcoins.core.protocol.dlc.models.DLCStatus
+import org.bitcoins.core.protocol.tlv.{
+  ContractInfoV0TLV,
+  DLCAcceptTLV,
+  DLCOfferTLV,
+  DLCSignTLV,
+  LnMessageFactory,
+  OracleAnnouncementV0TLV
+}
+import org.bitcoins.gui.contract.GlobalContractData
+import org.bitcoins.gui.dlc.DLCPaneModel
+import org.bitcoins.gui.dlc.dialog.{
+  AcceptOfferDialog,
+  BroadcastDLCDialog,
+  CreateDLCOfferDialog,
+  DLCDialogContainer,
+  SignDLCDialog,
+  ViewDLCDialog
+}
+import org.bitcoins.gui.util.GUIUtil
+import scalafx.beans.property.StringProperty
+import scalafx.geometry._
+import scalafx.scene.Parent
+import scalafx.scene.control.{
+  ContextMenu,
+  MenuItem,
+  TableColumn,
+  TableView,
+  TextField
+}
+import scalafx.scene.layout._
+
+import java.io.File
+import java.nio.file.Files
+import java.text.SimpleDateFormat
+import java.util.Date
+import scala.util.{Failure, Success}
+
+class ContractGUI(glassPane: VBox) {
+
+  def fetchStartingData(): Unit = {
+    // Nothing to fetch for state yet...
+  }
+
+  private[gui] lazy val model = new ContractGUIModel()
+
+  private lazy val addEventTF = new TextField {
+    styleClass += "title-textfield"
+    promptText = "New Event Hex"
+    onKeyTyped = _ => {
+      val event = model.addEvent(this.text.value.trim)
+      event match {
+        case Some(tup) =>
+          clearEventTF()
+          eventTableView.sort()
+          // Set focus on new item
+          eventTableView.getSelectionModel().select(tup)
+          // Show view
+          showCreateOfferPane(tup._1, tup._2)
+        case None =>
+      }
+    }
+  }
+
+  private def clearEventTF(): Unit = {
+    addEventTF.clear()
+  }
+
+  lazy val addEventHBox = new HBox {
+    styleClass += "small"
+    children = Seq(addEventTF)
+  }
+
+  private lazy val addContractTF = new TextField {
+    styleClass += "title-textfield"
+    promptText = "Contract Hex"
+    onKeyTyped = _ => {
+      val bool = onContractAdded(text.value.trim, None)
```

Comment:
nit: should use a more descriptive name than `bool`

Reply:
`isContractAdded`?